src/cmr/concepts/acl.js

```diff
@@ -0,0 +1,36 @@
+import AclConcept from './aclConcept'
+
+export default class Acl extends AclConcept {
+  /**
+   * Instantiates an Acl object
+   * @param {Object} headers HTTP headers provided by the query
+   * @param {Object} requestInfo Parsed data pertaining to the Graph query
+   * @param {Object} params GraphQL query parameters
+   */
+
+  constructor(headers, requestInfo, params) {
+    super('acls', headers, requestInfo, params)
+  }
+
+  /**
+   * Parse and return the array of data from the nested response body
+   * @param {Object} jsonResponse HTTP response from the CMR endpoint
+   */
+  parseJsonBody(jsonResponse) {
+    const { data } = jsonResponse
+
+    const { items } = data
+
+    return items
+  }
+
+  /**
+   * Query the CMR UMM API endpoint to retrieve requested data
```

Comment:
"CMR UMM API" does not mean much to me as a CMR developer, unless you're trying to say collections, so maybe replace UMM with ACL. These are different end points and otherwise I would think your talking about /search/ and not access control